semantic release action error

```
Run bunx semantic-release
  bunx semantic-release
  shell: /usr/bin/bash -e {0}
  env:
    GITHUB_TOKEN: ***
    NPM_TOKEN: ***
[semantic-release]: node version >=[2](https://github.com/apa-developer/apa-ui/actions/runs/12123425417/job/33798883042#step:6:2)0.8.1 is required. Found v18.20.5.

See https://github.com/semantic-release/semantic-release/blob/master/docs/support/node-version.md for more details and solutions.
Error: Process completed with exit code 1.
```

--- a/.github/workflows/branches-on-push.yml
+++ b/.github/workflows/branches-on-push.yml
@@ -14,6 +14,11 @@ jobs:
             - name: Checkout code
               uses: actions/checkout@v4
 
+            - uses: actions/setup-node@v4
+              with:
+                  node-version: '20.x'
+                  registry-url: 'https://registry.npmjs.org'
+
             - name: Setup bun
               uses: oven-sh/setup-bun@v2
 
@@ -27,4 +32,5 @@ jobs:
               env:
                   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
                   NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
               run: bunx semantic-release

